Source/CarthageKit/GitHub.swift

```diff
@@ -97,7 +97,7 @@ public struct GitHubRepository: Equatable {
 }
 
 public func ==(lhs: GitHubRepository, rhs: GitHubRepository) -> Bool {
-	return lhs.owner == rhs.owner && lhs.name == rhs.name
+	return lhs.owner.lowercaseString == rhs.owner.lowercaseString && lhs.name.lowercaseString == rhs.name.lowercaseString
```

Comment:
It's usually preferable to use a case-insensitive comparison operation (like `String.caseInsensitiveCompare`). But since the strings here are limited to ASCII, I'm not sure if it matters.

Anyone else in @Carthage/carthage have an opinion?